libs/core/device-manager/reducers/device-manager.reducer.ts

```diff
@@ -104,5 +107,17 @@ export const deviceManagerReducer = createReducer<DeviceManagerState>(
           activeDeviceId: undefined,
         }
       })
+      .addCase(openSelectDeviceDrawer, (state) => {
```

Comment:
⚪ Neutral
maybe toggler? less code

```
toggleSelectDeviceDrawer(true)
toggleSelectDeviceDrawer(false)
```

Reply:
I think u are suggesting method to simply update state, not a toggler. Toggles changes value to the opposite.

Reply:
Yes, you understood the essence of my suggestion. However, remember that you don't have to consider it at all, as the change is great.